queries/heex/injections.scm

```diff
@@ -0,0 +1,7 @@
+; Directives are combined to support do blocks
+(directive (expression_value) @elixir @combined)
```

Comment:
`@combined` isn't currently working as expected, it will break other modules like selection, folding, etc.